docs/configuration/ssl.md

```diff
@@ -56,6 +57,16 @@ If you decide to obtain a CA signed certificate, you can import that certificate
 
 The `certs:remove` command only works on app-specific certificates. It will `rm` the app-specific tls directory, rebuild the nginx configuration, and reload nginx.
 
+### Certificate export
```

Comment:
"Showing the certificate" would be a better title :)

Reply:
In addition, should we add a warning about this function as it exposes the internal certificate to anyone with access to the command?

Reply:
I would be more worried about database access and that I can't pin users to apps. Not that they can download the SSL cert. 

A rogue user that wants the private key could always upload a custom nginx sigil
that exposes the certificate directory.

Reply:
Seems legit.